widgets/htdocs/alphafold/controllers/molstarController.js

```diff
@@ -32,7 +33,13 @@ export class MolstarController {
     const uniprotId = alphafoldId.match(parsingRegex)[1];
     const url = `${alphafoldPredictionEndpoint}/${uniprotId}`;
 
-    const alphafoldPredictionEntries = await fetch(url).then(response => response.json());
+    const alphafoldPredictionEntriesResponse = await fetch(url);
+    if (alphafoldPredictionEntriesResponse.status === 404) {
```

Comment:
the status code 404 is wrong, isn't it? we might get a 404 when a request is broken or wrong endpoint or anything else we entered in the URL is wrong, it does mean we have no data but the URL is wrong. Is that confirmed from uniprot that they will send a status code 404 when there is no data found. 

To me it feels like we should check the response and if it is empty ({}) then we show this error message.

Reply:
> Is that confirmed from uniprot that they will send a status code 404 when there is no data found.

It's Alphafold, not Uniprot; and yes, they seem to have a well-behaved api in this respect:

![image](https://user-images.githubusercontent.com/6834224/193553494-3829ac7b-0761-411d-8bbe-d926c3792e10.png)

![image](https://user-images.githubusercontent.com/6834224/193553832-f7f83c99-31cc-4f0c-a0ea-82c7f4444e0f.png)

I think we can trust them on this.

Reply:
But this URL also give the same status code 404 but the response is different. It looks like they are sending same status code (404) but the response is different. The question again whether its better to check the response or just the status code?

https://alphafold.ebi.ac.uk/api/predictionsdfsd/P51587


![image](https://user-images.githubusercontent.com/6040010/193555680-89965820-9128-4b10-b354-2587ee777ead.png)


![image](https://user-images.githubusercontent.com/6040010/193555784-8600adff-0c9e-44b8-b25d-0c99158f1c5c.png)

Reply:
> But this URL also give the same status code 404

`https://alphafold.ebi.ac.uk/api/predictionsdfsd/P51587` is a url generated according to a broken schema. The schema according to which we are querying the alphafold api is defined on lines 31-34 above. If the schema is wrong, then the alphafold widget will stop working altogether, for all molecules; and we will quickly be notified by annoyed users.

Reply:
Based on @azangru , we are relying on the alphafold to send us the 404 status code when there is no data and we can rely on this for being stable. That is instead of checking the response is empty, we are happy to just check the status code 404. My concern is if the URL is broken or alphafold send us the 404 status code for something else then we might have to revise this approach to check the response. 

For now the solution is working fine and i am happy with the changes.